samples/highcharts/dragdrop/drag-xrange/demo.js

```diff
@@ -8,35 +13,76 @@ Highcharts.chart('container', {
         text: 'Highcharts draggable xrange demo'
     },
 
+    tooltip: {
+        headerFormat: '<span style="font-size: 10px">{point.yCategory}</span><br/>',
+        pointFormat: '{point.name}'
+    },
+
     plotOptions: {
         series: {
+            minPointLength: 5, // Always show points, even when resized down
             dataLabels: {
-                enabled: true
+                enabled: true,
+                format: '{point.name}'
+            },
+            dragDrop: {
+                draggableX: true,
+                draggableY: true,
+                dragMinY: 0,
+                dragMaxY: 2,
+                dragMinX: Date.UTC(2014, 10, 15),
+                dragMaxX: Date.UTC(2015, 0, 10),
+                liveRedraw: false,
+                groupBy: 'groupId' // Group data points with the same groupId
             },
             point: {
                 events: {
+                    dragStart: function (e) {
+                        var status = 'Drag started at page coordinates ' +
+                                e.pageX + '/' + e.pageY;
+                        if (e.updateProp) {
+                            status += '. Updating ' + e.updateProp;
```

Comment:
You could consider doing this conditional within the assignment and initialization of status. Would likely be shorter.